### DISCUSSION TITLE
azerothcore-all-stackables-200 sql

### DISCUSSION THREAD

**Discussion by gudd101** [azerothcore/azerothcore-wotlk#19127] [General]
Hi, I have no idea what i have to do, trying to run azerothcore-all-stackables-200 using putty. I have placed this sql file on my module folder. I go to maria then acore_world then I type Use All_Stackables_200_Up.sql and it gives me ERROR 1049 (42000): Unknown database 'All_Stackables_200_Up.sql' . When i use All_Stackables_200_Up.sql it give me another promt , I type exit; ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'All_Stackables_200_Up.sql

If any one know how to run this file correctly please help and provide the steps I have to do to run this sql file correctly on the server.

**Comment by ryankennedy85** [comment]
If you’re still having issues, or for anyone in the future, try copying and pasting the query into the query field and running it rather than running it from the menu. The file itself is just a list of “update item_template”s and can’t be opened like that.

**Comment by skirmess** [comment]
I used this to make everything that's currently stackable, stack to 1000. I haven't noticed any issues with it. If you want to stack only to 200, change the two 1000 to 200 in the last line. I did copy the query from [issue #2](https://github.com/AsgavinYT/azerothcore-all-stackables-200/issues/2) of the module.

```mysql> use acore_world;

CREATE TABLE item_template_backup (
entry int unsigned NOT NULL DEFAULT '0',
stackable int DEFAULT '1',
PRIMARY KEY (entry)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT='Item Stack size backup';

INSERT INTO acore_world.item_template_backup (entry,stackable) SELECT entry,stackable FROM acore_world.item_template WHERE acore_world.item_template.stackable > 1;

UPDATE acore_world.item_template SET stackable=1000 WHERE stackable > 1 and stackable < 1000;
